aten/src/ATen/native/BlasKernel.cpp

```diff
@@ -180,18 +181,22 @@ void gemv(char trans, int64_t m, int64_t n, scalar_t alpha, scalar_t *a, int64_t
     return;
   }
 
+  using accscalar_t = at::acc_type<scalar_t, false>;
```

Comment:
How about using `opmath_t` consistently?

Reply:
Sounds good, have followed this way.